composer.json

```diff
@@ -4,7 +4,7 @@
     "minimum-stability": "stable",
     "prefer-stable": true,
     "require": {
-        "php": "~8.1",
+        "php": "~8.3",
```

Comment:
Pour rappel (installation de la version php dans l'app scalingo) https://doc.scalingo.com/languages/php/start#:~:text=more%20recent%20version.-,Select%20a%20Version,-The%20default%20PHP